### PR TITLE
Add blogpost about what we have done in SG the past year

### PR DESCRIPTION
**Pull request by jonasac**

### CODE REVIEW

File: _posts/2018-09-05-ship-shape.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## Decoupling Web & API
+
+
+The Shipping Guide is primarily an API, but in the beginning it also served a frontend. Some of the frontend features had been subsumed by other applications in our ecosystem, some were no longer in use and rest were stuff we wanted to keep around in a more up to date and modern fashion. 
+
+We decided move all frontend related code to a different deployable which would get its data from The Shipping Guide the same way our users does. Keeping the frontend and backend contexts separate like this enabled us to work faster on each deployable and gave us more confidence in cleaning up the frontend code without messing up stuff for the backend. 
+
+
+## Summary
```
Comment (jonasac) [inline _posts/2018-09-05-ship-shape.md]: Needs suggestions for a headline here, feels a bit homework oriented to call it summary

File: _posts/2018-09-05-ship-shape.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+All in all we are quite happy about where The Shipping Guide is now. There will always been things to improve in projects like this, but our focus on cleaning up the things that would let us move faster with more confidence have made those future cleanups seem less like extensive and more like a walk around the woods.
+
+Below are some interesting stats across some of the many releases of The Shipping Guide the past year.
+
+
+### Code smells
```
Comment (jonasac) [inline _posts/2018-09-05-ship-shape.md]: Elisabeth suggested that we add some texts to these images, maybe explain in plain english what they should be

File: _posts/2018-09-05-ship-shape.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Shipshape
+date: 2018-09-05 16:00:00 01:00
+category: code-and-design
+author:
+    - jonasac
```
Comment (jonasac) [inline _posts/2018-09-05-ship-shape.md]: Who else wants to be here?

File: _posts/2018-09-05-ship-shape.md (hunk 1 above)
Comment (pankajagrawal16) [reply _posts/2018-09-05-ship-shape.md]: lol..

File: _posts/2018-09-05-ship-shape.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Shipshape
+date: 2018-09-05 16:00:00 01:00
+category: refactoring
+author:
```
[the remark this answers is not in the document]
Comment (jonasac) [reply _posts/2018-09-05-ship-shape.md]: @pankajagrawal16 @shikhavyaghra @priyamittal @best-ankitagarwal @chandnaankit @elisabethirgens reply to this with yay if you are ok with having your name here

File: _posts/2018-09-05-ship-shape.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Shipshape
```
Comment (jonasac) [inline _posts/2018-09-05-ship-shape.md]: @pankajagrawal16 i feel like `Shipshape Shipping Guide` is giving away the pun, but then again our priorites should probably not be my pleasure in puns :)

`Shipshape Shipping Guide` sounds ok ? I'll try to come up with more elusive alternatives, but if i can't we can go with that.

Comment (pankajagrawal16) [reply _posts/2018-09-05-ship-shape.md]: hahaha ... yeah it does.. to me ShipShape Shipping guide looks better to catch attention on blog post list, but that might be my familiarity with SG. :p

File: _posts/2018-09-05-ship-shape.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (best-ankitagarwal) [reply _posts/2018-09-05-ship-shape.md]: @jonasac  is it possible to include all thumbsup as authors now
